autoware_launch/config/system/diagnostics/autoware-main.yaml

```diff
@@ -63,6 +63,7 @@ units:
     list:
       - { type: link, link: /autoware/vehicle }
       - { type: link, link: /autoware/system }
+      - { type: link, link: /autoware/planning }
```

Comment:
This PR modifies the diagnostics/ directory.
But it seems that TIER IV internally uses tier4_diagnostics directory, so please update it instead of this directory.